spec/puppet-syntax/manifests_spec.rb

```diff
@@ -78,19 +114,34 @@
         context 'Puppet >= 3.2' do
           it 'should pass with future option set to true on future manifest' do
             files = fixture_manifests(['future_syntax.pp'])
-            res = subject.check(files)
+            output, has_errors = subject.check(files)
 
-            expect(res.size).to eq(0)
+            expect(output).to eq([])
+            expect(has_errors).to eq(false)
+          end
+        end
+        context 'Puppet >= 3.7' do
+          # Certain elements of the future parser weren't added until 3.7
+          if Puppet::Util::Package.versioncmp(Puppet.version, '3.7') >= 0
+            it 'should fail on what were deprecation notices in the non-future parser' do
+              files = fixture_manifests('deprecation_notice.pp')
+              output, has_errors = subject.check(files)
+
+              expect(output.size).to eq(1)
+              expect(output[0]).to match(/Node inheritance is not supported/)
+              expect(has_errors).to eq(true)
+            end
           end
         end
       else
         context 'Puppet <= 3.2' do
```

Comment:
The label `Puppet <= 3.2` is not quite accurate as this context will only be evaluated if the version is less than `3.2` (based on the `if` condition on line 113). Could you please change it to `Puppet < 3.2`?